spec/features/sale/update_a_sale_spec.rb

```diff
@@ -1,130 +0,0 @@
-require 'rails_helper'
```

Comment:
I removed the feature to edit a sale. Because I wanna keep this registry hard. Later the user may close that sale or create a new.